PayPal.xcodeproj/project.pbxproj

```diff
@@ -3184,6 +3184,7 @@
 				SUPPORTS_MACCATALYST = YES;
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = "$(inherited) DEBUG";
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				SWIFT_STRICT_CONCURRENCY = minimal;
```

Comment:
Should this be included in this PR?

Reply:
No. It snuck in there. I will get rid of it.

Reply:
Addressed here: 80ac172b81e04c24f3d9062ee5ee22e934b5546f